x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -7,6 +7,8 @@ var security = (function () {
     var processor = require("processor");
     var winlogbeat = require("winlogbeat");
 
+    // Logon Types
+    // https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/basic-audit-logon-events
```

Comment:
:heart: Thank you for putting in links to where the mappings come from.

Reply:
> @janniten This is awesome.
> 
> I did get some errors when I tried to make the golden files.
> 
> ```postscript
>     --- FAIL: TestSecurity/security-windows2012_4770.evtx (0.02s)
>         testing_windows.go:70: failed in processor.javascript: failed in process function: TypeError: Cannot read property '2' of undefined at ..\config\winlogbeat-security.js:1356:38(17) while processing event:
> ...
> ```
> 
> It doesn't look like you don't have events 4770, 4771 in the eventActionTypes object. Could you add those?

Added and tested :)
![image](https://user-images.githubusercontent.com/33020901/78695306-384fc680-78fe-11ea-9a7b-a18bc00e8d6c.png)